### PR TITLE
Add more digits to the delta of openDX

### LINKED ISSUE #88: Use scientific notation for the delta representation
I'm interested in using GridDataFormats to generate input OpenDX grid for APBS calculations.
For box dimensions of (90, 90, 150) with 257 data points in each direction.
The APBS will give the header:

```
object 1 class gridpositions counts 257 257 257
origin -4.500000e+01 -4.500000e+01 -7.500000e+01
delta 3.515625e-01 0.000000e+00 0.000000e+00
delta 0.000000e+00 3.515625e-01 0.000000e+00
delta 0.000000e+00 0.000000e+00 5.859375e-01
object 2 class gridconnections counts 257 257 257
```

The GridDataFormats will give the header

```
object 1 class gridpositions counts  257 257 257
origin -45.000000 -45.000000 -75.000000
delta  0.351562 0.000000 0.000000
delta  0.000000 0.351562 0.000000
delta  0.000000 0.000000 0.585938
object 2 class gridconnections counts  257 257 257
```

Since the GridDataFormats will give one less digit in the delta section, the OpenDX grid is rejected by the APBS. 

This easily be solved by changing
https://github.com/MDAnalysis/GridDataFormats/blob/be6132ac13041390a880061e4e873044b6c29573/gridData/OpenDX.py#L244
to 

`file.write('delta '+self.ndformat(' {:.6e}').format(*delta) +'\n')`

but I'm not sure if this will break other things.

### PR DESCRIPTION
**Pull request by xiki-tempula**
Fix #88

### CODE REVIEW

File: gridData/OpenDX.py [hunk 1]
```diff
@@ -241,7 +241,9 @@ def write(self, stream):
         self._write_line(stream, 'origin %f %f %f\n' % tuple(self.origin))
         for delta in self.delta:
             self._write_line(
-                stream, ('delta '+self.ndformat(' %f')+'\n') % tuple(delta))
+                stream, ('delta ' +
```
Comment (IAlibay) [inline gridData/OpenDX.py:244]: Do we need an extra test to check that your use case is now covered?

**Review by IAlibay — CHANGES_REQUESTED** [review]
If you update against master CI should now be green (hopefully?)

I'll let @orbeckst do the final review here (since I'm technically new to this repo).

I _think_ you need to add yourself to AUTHORS?

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thanks for the PR, @xiki-tempula ; please add
- CHANGELOG
- AUTHORS
- small test that checks that the guaranteed number of digits is saved (and read) — maybe just use your example
- add entry to docs for OpenDX stating how we are saving (for instance, for PyMOL we have an entry on precision)

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Thanks @xiki-tempula !

@IAlibay do the latest changes address your concerns?

**Review by IAlibay — APPROVED** [review]
lgtm, just the one thing about the docs authorship

**Pull request merged**